boards/m5stack_core2/axp202c.py

```diff
@@ -0,0 +1,820 @@
+'''

```

Comment:
Duplicated.
Also under `lilygo_twatch_2020/axp202c.py`.
Should probably go under `generic/driver`.
Is it very different from [axp192](https://github.com/lvgl/lv_binding_micropython/blob/master/driver/generic/axp192.py)?